routes.go

```diff
@@ -195,8 +569,8 @@ func BadgeHandler(c *gin.Context) {
 	switchedFromYesToNo := false
 
 	badgeChoice := c.PostForm("badge")
-	if badgeChoice != user.Badge {
-		if user.Badge == "yes" && badgeChoice == "no" {
+	if (badgeChoice == "yes") != user.Badge {
```

Comment:
is this intentional? i think we had badges as yes/no/null before. did you just make them yes/no?

Reply:
yes, since we're asking up front with the food information I thought there wouldn't be a need for a null option, you think we should keep it with the null?

Reply:
oh i see it. your way is even better -- make them pick up front